proto/authenticator.gen.go

```diff
@@ -54,13 +54,17 @@ const (
 	IntentName_sessionAuthProof      IntentName = "sessionAuthProof"
 	IntentName_feeOptions            IntentName = "feeOptions"
 	IntentName_signMessage           IntentName = "signMessage"
+	IntentName_signTypedData         IntentName = "signTypedData"
 	IntentName_sendTransaction       IntentName = "sendTransaction"
 	IntentName_getTransactionReceipt IntentName = "getTransactionReceipt"
 	IntentName_federateAccount       IntentName = "federateAccount"
 	IntentName_removeAccount         IntentName = "removeAccount"
 	IntentName_listAccounts          IntentName = "listAccounts"
 	IntentName_getIdToken            IntentName = "getIdToken"
 	IntentName_adoptChildWallet      IntentName = "adoptChildWallet"
+	IntentName_getAdopter            IntentName = "getAdopter"
```

Comment:
I guess we're adding the parent/child stuff in this PR as well..? no biggie, just noting

Reply:
what does "getAdopter" mean btw..? just wondering to see if I can propose perhaps a better name, or maybe once I understand the description it'll make more sense to me

Reply:
It simply returns the universal wallet's address that adopted the wallet, used by the client to know whether the embedded wallet is adopted (and by whom.)

The reason it's included in this PR too is because this is already merged in go-sequence but wasn't yet updated in this repo